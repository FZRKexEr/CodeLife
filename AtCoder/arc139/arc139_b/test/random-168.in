100
59 15 12 8 18 19
99 20 9 5 18 10
24 16 20 9 10 1
74 16 7 14 15 9
85 6 4 1 17 15
81 11 7 18 14 6
25 11 3 8 5 11
94 12 4 4 4 20
5 6 11 14 6 5
71 1 4 15 16 11
27 7 16 6 14 16
44 4 5 14 3 2
90 8 9 13 4 1
82 9 12 18 6 18
68 7 9 7 19 15
14 20 18 7 4 1
49 6 3 15 16 15
67 4 12 2 3 20
67 9 6 18 2 17
37 8 7 2 9 3
100 2 20 17 19 13
53 19 15 6 18 8
4 17 11 11 19 2
94 18 11 10 9 16
62 17 8 12 19 14
40 19 4 19 7 16
98 10 9 5 11 10
86 15 5 10 4 19
96 3 4 10 2 13
95 5 13 2 18 17
15 13 10 16 9 20
44 16 11 7 9 20
11 6 14 8 13 9
93 2 19 13 19 2
7 1 8 8 6 16
57 12 6 9 5 5
65 10 4 10 3 6
83 19 20 8 9 12
16 13 12 19 2 20
58 15 6 17 2 10
20 2 2 13 18 13
37 3 16 13 3 8
2 2 10 20 12 16
54 10 14 4 10 17
7 16 16 1 20 6
50 12 19 3 15 2
28 8 11 18 5 11
56 3 7 7 12 4
58 1 13 13 13 14
33 13 13 7 1 3
28 12 2 15 19 18
41 15 18 20 1 8
69 7 17 20 14 14
38 6 7 14 19 19
59 13 1 11 18 12
44 5 19 15 4 12
7 1 18 20 12 11
47 16 12 10 10 7
14 10 15 14 16 4
93 10 19 16 20 8
93 11 12 2 4 15
75 12 10 18 17 15
100 20 5 19 7 17
82 13 8 3 17 19
35 17 18 18 19 18
74 12 19 11 20 5
30 8 20 19 17 9
100 17 16 20 7 14
34 18 15 18 20 14
25 14 9 4 18 5
30 5 16 12 17 14
79 8 12 9 4 12
81 19 5 11 2 1
7 13 4 4 18 3
10 18 18 10 7 17
19 2 18 20 11 4
50 8 20 20 11 1
44 3 17 13 7 20
34 18 8 11 8 15
62 2 15 6 10 7
53 18 6 6 20 8
80 15 17 13 15 19
52 4 12 8 6 7
46 15 3 10 3 13
54 10 1 19 6 6
49 1 10 6 5 19
2 11 13 7 12 10
21 8 17 20 1 19
32 6 4 5 8 4
11 18 9 6 7 10
48 9 12 4 1 17
65 3 10 2 10 18
17 13 5 2 5 16
53 20 4 2 13 8
31 11 9 17 18 17
23 8 12 15 13 2
22 7 18 1 18 6
6 7 17 7 18 1
66 3 3 19 10 14
38 7 15 7 9 8
