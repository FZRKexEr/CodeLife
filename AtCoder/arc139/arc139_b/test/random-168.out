89
97
37
84
85
74
21
68
70
232
54
19
10
72
128
98
143
54
64
19
65
44
44
68
107
52
108
68
64
127
41
45
53
173
42
52
34
72
59
43
130
29
12
57
7
19
39
32
134
51
74
116
76
114
149
32
84
57
72
164
42
127
35
114
37
70
64
56
52
31
96
71
9
15
100
24
53
74
38
38
86
96
34
19
54
121
14
99
32
22
17
118
13
52
53
60
10
42
220
32
